bin/3llo

```diff
@@ -24,8 +43,6 @@ $container.register(
 user = Tr3llo::API::User.find($container.resolve(:configuration).user_id)
 $container.register(:user, user)
 
-Tr3llo::Presenter::HelpPresenter
-  .new($container.resolve(:interface))
-  .print!
+print_help(Tr3llo::Presenter::FLAVOUR_HELP_INDENTED)
```

Comment:
So a help menu outside of the programs normal use (since it's completely interactive) would be a different explanation of use.

And in this case it should explain to set those environment variables.
That way you get the nice error and the explanation.

##### An aside, a good future improvement would be to have flags on the command line for the environment variables.

Reply:
I added a short help text for the env variables, but figured that it might be nice to get an overview over the interactive functionality as well.